poetry_scripts.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+
+def format_code():
+    os.system("poetry run black .")
+    os.system("poetry run isort .")
+    os.system("poetry run autoflake .")
```

Comment:
same idea here, we could make sure the same flags are consistent across different locations:
- this script
- pre-commit config
- poetry tooling config
- CI tests

Reply:
I have verified that all these different executions make use of the settings defined in `pyproject.toml`.

Reply:
thanks